coral/src/app/components/ComplexNativeSelect.tsx

```diff
@@ -50,6 +50,7 @@ function ComplexNativeSelect<T>(props: ComplexNativeSelectProps<T>) {
 
   const nativeSelectProps = omit(
     props,
+    "placeholder",
```

Comment:
Passing `placeholder` while we still have the old placeholder workaround results in tests finding *two* placeholders